pkgs/development/python-modules/pyqt/6.x.nix

```diff
@@ -0,0 +1,138 @@
+{ lib
+, buildPythonPackage
+, isPy27
+, fetchPypi
+, pkg-config
+, dbus
+, lndir
+, dbus-python
+, sip
+, pyqt6-sip
+, pyqt-builder
+, qt6Packages
+, withConnectivity ? false
+, withMultimedia ? false
+, withWebKit ? false
+, withWebSockets ? false
+, withLocation ? false
```

Comment:
This is highly problematic in python packages. Is pyqt6 used in propagatedBuildInputs? If so we need to run on all options by default and best remove them.

Reply:
It could be, can remove the options.

Do I need to remove them from the 5.x package?

Reply:
Looks like have to keep `withWebEngine` off by default - see the 5.x version:

![image](https://user-images.githubusercontent.com/782440/185770228-ec34daf7-a8cc-41a9-86c3-0ac296c57a40.png)

Reply:
> Do I need to remove them from the 5.x package?

Thats a topic for another PR.



> Looks like have to keep `withWebEngine` off by default - see the 5.x version:

The comment is only there because of the issue I described earlier. Also only adding a comment is not that great because it can be easily missed in a review.

Reply:
why exactly is this problematic? Building with webengine by default may be a bit overkill for applications that don't need it, as webengine is a pretty heavy dependency both build time wise, and on disk.

Reply:
Updated the ones that work to true, added comments otherwise.